apps/web/src/pages/settings/user-profile-page/UserProfileForm.tsx

```diff
@@ -0,0 +1,105 @@
+import { FC } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import * as capitalize from 'lodash.capitalize';
+import { Button, errorMessage, Input, successMessage } from '@novu/design-system';
+import type { IUserEntity } from '@novu/shared';
+
+import { css } from '../../../styled-system/css';
+import { ProfileImage } from '../../../components/shared';
+import { useUpdateUserProfile } from '../../../api/hooks';
+
+interface IUserProfileForm {
+  firstName: string;
+  lastName: string;
+}
+
+interface IUserProfileFormProps {
+  currentUser?: IUserEntity | null;
+}
+
+const formInputStyles = css({
+  minWidth: '18.75rem',
+  '& label': { marginTop: '0 !important' },
+  '& .mantine-TextInput-error': { position: 'absolute', marginTop: '0.5rem' },
```

Comment:
render errors on position absolute, otherwise, the form is jumping like:

![Screenshot 2024-04-08 at 18 39 20](https://github.com/novuhq/novu/assets/2607232/331b0824-c6db-4262-bad7-e6a561c17a47)